sdk/storage/azure-storage-file-datalake/azure/storage/filedatalake/aio/_path_client_async.py

```diff
@@ -185,8 +185,16 @@ async def _create(self, resource_type, content_settings=None, metadata=None, **k
             The timeout parameter is expressed in seconds.
         :return: Dict[str, Union[str, datetime]]
         """
+        lease_id = kwargs.pop('lease_id', None)
+        lease_duration = kwargs.pop('lease_duration', None)
```

Comment:
You can use `get` instead of `pop` so you don't pull the values out and therefore don't need to pass them individually into `_create_path_options`. `get` also accepts a default Value like `pop`

Reply:
Thanks for this! I looked around to try to find the design pattern for what I wanted to achieve, and that's exactly what I was looking for :)